src/qibo/models/dbi/double_bracket.py

```diff
@@ -20,6 +25,26 @@ class DoubleBracketGeneratorType(Enum):
     # TODO: add double commutator (does it converge?)
 
 
+class DoubleBracketScheduling(Enum):
+    """Define the DBI scheduling strategies."""
+
+    hyperopt = hyperopt_step
+    """Use hyperopt package."""
+    grid_search = grid_search_step
+    """Use greedy grid search."""
+    polynomial_approximation = polynomial_step
+    """Use polynomial expansion (analytical) of the loss function."""
+
+class DoubleBracketCostFunction(Enum):
```

Comment:
```suggestion
class DoubleBracketCostFunctionType(Enum):
```

because auto() is used, this means throughout the name should be changed

Reply:
Not really, you can put any name.
I would even shorted the name and opt for something like this.
```suggestion
class DoubleBracketCost(Enum):
```